### PR TITLE
update Arg pybamm.Experiment.period

### PR DESCRIPTION
**Pull request by AndyLiuElysia**
# Description

The Arg in pybam.experiment has been updated. 
This is one of the issue listed in #3392 

## Type of change

Please add a line in the relevant section of [CHANGELOG.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CHANGELOG.md) to document the change (include PR #) - note reverse order of PR #s. If necessary, also add to the list of breaking changes.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [ ] Bug fix (non-breaking change which fixes an issue)

# Key checklist:

- [x] No style issues: `$ pre-commit run` (or `$ nox -s pre-commit`) (see [CONTRIBUTING.md](https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md#installing-and-using-pre-commit) for how to set this up to run automatically when committing locally, in just two lines of code)
- [ ] All tests pass: `$ python run-tests.py --all` (or `$ nox -s tests`)
- [ ] The documentation builds: `$ python run-tests.py --doctest` (or `$ nox -s doctests`)

You can run integration tests, unit tests, and doctests together at once, using `$ python run-tests.py --quick` (or `$ nox -s quick`).

## Further checks:

- [ ] Code is commented, particularly in hard-to-understand areas
- [ ] Tests added that prove fix is effective or that feature works

### CODE REVIEW

**Comment by AndyLiuElysia** [comment]
this is part of the preparation for the April PyBamm training.

File: pybamm/experiment/experiment.py [hunk 1]
```diff
@@ -20,9 +20,9 @@ class Experiment:
     ----------
     operating_conditions : list[str]
         List of strings representing the operating conditions.
-    period : string, optional
+    period : str, optional
         Period (1/frequency) at which to record outputs. Default is 1 minute. Can be
-        overwritten by individual operating conditions.
+        overwritten by individual operating conditions. e.g. 30 minute, 1 hour.
```
Comment (kratman) [inline pybamm/experiment/experiment.py:25]: I don't think the 30 min part adds anything here, if anything it makes it more confusing. 
```suggestion
        overwritten by individual operating conditions.
```

**Review by kratman — CHANGES_REQUESTED** [review]

File: pybamm/experiment/experiment.py (hunk 1 above)
Comment (AndyLiuElysia) [reply pybamm/experiment/experiment.py]: Following the suggested change, I have removed it. 
Shall we add '_e.g. in second, minute, hour_' to describe the possible string values in period?

**Review by Saransh-cpp — APPROVED** [review]
Thanks, @AndyLiuElysia!

**Review by kratman — APPROVED** [review]

**Comment by valentinsulzer** [comment]
@all-contributors add @AndyLiuElysia for docs

**Pull request merged**